stl/inc/mutex

```diff
@@ -522,7 +521,21 @@ auto _Invoke_stored(tuple<_Types...>&& _Tuple)
     return _Invoke_stored_explicit(_STD move(_Tuple), index_sequence_for<_Types...>());
 }
 
+// STRUCT once_flag

```

Comment:
`once_flag` and `call_once` are depicted last in [mutex.syn]; any reason we can't relocate these while we're here?

Reply:
It looks like they are already located correctly -- the following condition_variable stuff comes from `<condition_variable>` in the standard.